### PR TITLE
feat(codesnippet/textarea): add experimental components to website

### LINKED ISSUE #438: Add Experimental Code Snippet to website
- [ ] Add Experimental component to website
- [ ] Update component status

### LINKED ISSUE #439: Add Experimental Text Area to website
- [ ] Add Experimental component to website
- [ ] Update component status

### PR DESCRIPTION
**Pull request by jnm2377**
Closes #439 
Closes #438 

#### Changelog

**New**
- 2 new Experimental components: Code Snippet, Text Area (included in Text Input)

### CODE REVIEW

File: src/content/experimental/text-input/code.md [hunk 1]
```diff
@@ -6,10 +6,20 @@ tabs: ['Code', 'Usage']
...
     name="Experimental Text Input"
-    component="text-input" 
+    component="text-input"
     variation="text-input"
     experimental="true"
     >
 </component>
+
+<component
+    name="Text Area"
```
Comment (alisonjoseph) [inline src/content/experimental/text-input/code.md]:
```suggestion
    name="Experimental Text Area"
```

File: src/content/experimental/code-snippet/code.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+label: Experimental
+title: Code Snippet
+tabs: ['Code', 'Usage']
+---
+
+<page-intro>**Code snippets** are small blocks of reusable code that can be inserted in a code file.
+variations:</page-intro>
+
+<component
+    name="Code Snippet"
```
Comment (alisonjoseph) [inline src/content/experimental/code-snippet/code.md]:
```suggestion
    name="Experimental Code Snippet"
```

File: src/content/experimental/code-snippet/code.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    component="code-snippet"
+    variation="code-snippet"
+    experimental="true"
+    >
+</component>
+<component
+    name="Inline Code Snippet"
```
Comment (alisonjoseph) [inline src/content/experimental/code-snippet/code.md]:
```suggestion
    name="Experimental Inline Code Snippet"
```

File: src/content/experimental/code-snippet/code.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    component="code-snippet"
+    variation="code-snippet--inline"
+    haslightversion="true"
+    experimental="true"
+    >
+</component>
+<component
+    name="Multi Line Code Snippet"
```
Comment (alisonjoseph) [inline src/content/experimental/code-snippet/code.md]:
```suggestion
    name="Experimental Multi Line Code Snippet"
```

**Comment by jnm2377** [comment]
@alisonjoseph made those changes you request. As for the space around the "Show More" button, that was actually a design change that happened after the initial PR request. The new specs are on here: https://github.com/IBM/carbon-components/pull/1447

**Pull request merged**